config.json

```diff
@@ -34,7 +34,7 @@
       "description": "Specifies the distro rid for Unix OS.",
       "valueType": "property",
       "values": [],
-      "defaultValue": "${OSRid}"
+      "defaultValue": "${OSRid}-${CPUArch}"
```

Comment:
> is how DistroRid was forced to be portable RID when doing portable build.

Okay, I can try to think about how I can put that into build.sh while still not going whole hog parsing arguments there.